common/swagger/v1/swagger.yaml

```diff
@@ -279,6 +288,10 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/Row'
+      requestPredictionIntervals:
+        type: boolean
+        description: |
+          If set to `true`. The scorer will try to fill field `predictionIntervals` in response if it is supported.
```

Comment:
Should there be a comma? Like this
> If set to `true`, the scorer will try to fill field `predictionIntervals` in response if it is supported.

Reply:
thx, updated https://github.com/h2oai/dai-deployment-templates/pull/351/commits/f83117d0816b8434fc174e4cb0e5641375c47d4a